saleor/celeryconf.py

```diff
@@ -39,5 +39,11 @@ def setup_celery_logging(loglevel=None, **kwargs):
     ],
     related_name="saleor3_17",
 )
+app.autodiscover_tasks(
+    packages=[
+        "saleor.order.migrations.tasks",
+    ],
+    related_name="saleor3_13",
```

Comment:
I think it's worth to change the name of that file, as to not make ourselfs confused when we will be tracking running migrations tasks.

Reply:
Discussed with cloud team. It is okay with current form